code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -98,12 +98,21 @@
 	if(update_icon)
 		mannequin.update_icon()
 
-/datum/preferences/proc/update_preview_icon()
+/// Updates character mannequin without rebuilding the inventory or overlays of the mannequin.
+/datum/preferences/proc/refresh_character_preview()
+	var/mob/living/carbon/human/dummy/mannequin/mannequin = get_mannequin(client_ckey)
+	if(mannequin)
+		update_character_preview_mannequin(new /mutable_appearance(mannequin))
+
+/// Updates character preview map.
+/// If you don't need to update renders, use [/datum/preferences/proc/update_character_preview_background].
+/datum/preferences/proc/update_character_preview_map()
```

Comment:
Why rename this? It's confusing; this proc doesn't actually do anything with the map control that any of the other split out ones don't do; the main thing it does is the expensive step of deleting and rebuilding the mannequin inventory.

The number of procs involved in this whole business feels needlessly confusing.